src/components/textfield/textfield.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextfieldLabel } from './../typography/typography';
+
+const StyledLabel = styled(TextfieldLabel)`
+  display: block;
+  margin-bottom: 4px;
+
+  &.error {
+    font-weight: 700;
+    color: #ff0d35;
+  }
+`;
+
+interface Textfield {
+  errorMessage: string;
+  label: string;
+}
+
+const Textfield = ({ label, errorMessage }) => {
+  const StyledInputText = styled.input.attrs({
+    type: 'input',
+  })`
+    width: 320px;
+    height: 48px;
+
+    padding-left: 16px;
+    padding-right: 16px;
+
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: -0.01em;
+    color: #202840;
+
+    border: 2px solid;
+    border-color: ${errorMessage ? '#FF0D35' : '#f7f8fa'};
+    background: #f7f8fa;
+  `;
+
+  return (
+    <label>
+      <StyledLabel>{label}</StyledLabel>
+      <StyledInputText />
+      {errorMessage && (
+        <StyledLabel className="error">{errorMessage}</StyledLabel>
+      )}
```

Comment:
This is almost right but the spacings are slightly off.
![image](https://user-images.githubusercontent.com/7970458/142615547-b380e595-70af-40de-a154-996e3f503090.png)

I suggest to **not re-use the `StyledLabel` for the error message**. Instead introduce a new `StyledErrorMessage` (or similar) that also uses the new `TextXS`. The input label has a `margin-bottom` of 4px, whereas the error message needs to have a `margin-top` of 4px. Also the font-weight and the color differ. These are enough reasons to create a new styled component for this. Mostly the margin though. Also - I've learned - it's rather unusual to use classes with styled components.